common/app/views/fragments/inlineJSNonBlocking.scala.html

```diff
@@ -24,9 +24,6 @@
 
 // ************* ANALYTICS *************
 
-// Ophan pageview ID and browser ID are needed by Google Analytics, which runs just after this script tag
-@InlineJs(ophanConfig().body, "ophanConfig.js")
```

Comment:
Are you certain that this is only doing work related to GA? We would not want to lose Ophan tracking for this page.

Reply:
Not sure if this code was just added for GA or it is used by Ophan tracking. @SHession could you confirm if this line of code is needed for Ophan tracking?

Reply:
It's difficult for me to say if this is needed for tracking as I'm not familiar with this codebase. However, it does look Ophan and not GA related and I would be inclined to keep it unless we are confident it won't impact traffic.

Reply:
Re-added in 491f0d916a0c9ebdfcfe5a5e5ad122cc5137b9a1